jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -41,10 +41,42 @@ class AttentionLayout(Enum):
   BTNH = 0
   BNTH = 1
 
+class MaskType(Enum):
+  NO_MASK = 0
+  PADDING = 1
+  CAUSAL = 2
+  PADDING_CAUSAL = 3
+  ALIBI = 4
+
+def convert_mask_type_to_string(mask_type: MaskType) -> str:
```

Comment:
I think this is identical to `mask_type.name`.

Reply:
fixed